packages/headless/src/controllers/core/status/headless-core-status.ts

```diff
@@ -38,12 +38,12 @@ export interface CoreStatusState {
 }
 
 /**
- * Creates a `CoreStatus` controller instance.
+ * Creates a `SearchStatus` controller instance.
  *
  * @param engine - The headless engine.
- * @returns A `CoreStatus` controller instance.
+ * @returns A `SearchStatus` controller instance.
  * */
-export function buildCoreStatus(engine: CoreEngine): CoreStatus {
+export function buildCoreSearchStatus(engine: CoreEngine): SearchStatus {
```

Comment:
So I guess this point is contentious right? 
`CoreSearchStatus` or `CoreStatus` ?

Reply:
no it will just `CoreStatus`